quickstarts/observability-as-code/chef/config.yml

```diff
@@ -0,0 +1,45 @@
+id: ad40ccc2-f7ef-44bf-b961-07813088fce0
+
+# Name of the quickstart (required)
+name: chef
+
+# Displayed in the UI (required)
+title: Chef
+
+# Long-form description of the quickstart (required)
+description: |
```

Comment:
More detail here would be helpful for the user to understand the value of this quickstart.